src/django/api/reports/duplicates_merged.sql

```diff
@@ -0,0 +1,4 @@
+SELECT to_char(a.created_at, 'YYYY-MM') as month, COUNT(*) as duplicates_merged
+FROM api_facilityalias a
+WHERE reason = 'MERGE'
+GROUP BY month;
```

Comment:
We should include `ORDER BY month` so this is sorted like our other reports.

Reply:
Updated in  44a9a1f